common/src/main/java/com/viaversion/viaversion/protocols/protocol1_16to1_15_2/packets/EntityPackets.java

```diff
@@ -201,7 +200,7 @@ public void registerMap() {
                 map(Type.UNSIGNED_BYTE); //  Gamemode
                 handler(wrapper -> {
                     wrapper.write(Type.BYTE, (byte) -1); // Previous gamemode, set to none
-                    wrapper.write(Type.STRING_ARRAY, WORLD_NAMES); // World list - only used for command completion
+                    wrapper.write(Type.STRING_ARRAY, Via.getConfig().get1_16WorldNamesMap().values().toArray(new String[0])); // World list - only used for command completion
```

Comment:
Remove this, as it is basically useless with servers providing their own completion prior to the change

Reply:
Right, so clients don't really care about it being correct, since it's only use is for the specific brigadier `world` parser?

Removed.